torch/_inductor/lowering.py

```diff
@@ -1619,11 +1619,43 @@ def inner(
     return inner
 
 
-empty = register_lowering([torch.empty, aten.empty])(tensor_constructor(0))
 zeros = register_lowering([torch.zeros, aten.zeros])(tensor_constructor(0))
 ones = register_lowering([torch.ones, aten.ones])(tensor_constructor(1))
 
 
+@register_lowering([torch.empty, aten.empty])
+def empty(
```

Comment:
are we able to use the `empty_strided` to implement this to avoid code dup?

Reply:
Done.